.github/workflows/check-sorbet-typing-mode.yml

```diff
@@ -0,0 +1,59 @@
+name: Sorbet
+on: # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [main]
+    types: [opened, synchronize, reopened]
```

Comment:
I'd probably just set it to run on pull request, those activities are the default IIRC

```suggestion
```